### DISCUSSION TITLE
How to add/enable a new language for highlighting?

### DISCUSSION THREAD

**Discussion by finikm** [nuxt/content#1874] [Q&A]
I am trying to highlight some `.proto` files, but regardless of what I try, I haven't manage to do it. 

While they are supported by Shiki (https://github.com/shikijs/shiki/blob/main/docs/languages.md), I cannot use `proto` in `highlight.preload`. Doing some digging, I saw that Nuxt Content v2 uses [shiki-es](https://github.com/unjs/shiki-es), and Protobuf is not in its set of `BUNDLED_LANGUAGES`. I came across a similar [issue](https://github.com/nuxt/content/issues/1595), which while it's resolved, it didn't help. 

Any ideas are welcomed :)

Thanks

**Comment by Zerro97** [comment]
Have you tried configuring the nuxt.config.ts for syntax highlight?

```javascript
  content: {
    highlight: {
      preload: [
        'c',
        'cpp',
        '<any language supported by Shiki>'
      ],
  }
}
```

**Reply by finikm** [reply]
Indeed I tried, I get typescript error that the language is not a valid option, and if I try to bypass it (i.e. `preload: ['proto'] as any`) I get a runtime error.
